setup.py

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env python
+from codecs import open
+from os.path import realpath, dirname, join
+from setuptools import setup, find_packages
+import sys
+import re
+
+DISTNAME = 'pymc3_experimental'
+DESCRIPTION = "Experimental Modules of pymc3 (Probabilistic Programming in Python: Bayesian Modeling and Probabilistic Machine Learning with Theano)"
+AUTHOR = 'PyMC Developers'
+AUTHOR_EMAIL = 'pymc.devs@gmail.com'
+URL = "http://github.com/pymc-devs/pymc3-experimental"
+LICENSE = "Apache License, Version 2.0"
+
+classifiers = ['Development Status :: 5 - Production/Stable',
```

Comment:
should probably not have this label :)

Reply:
Thanks for pointing out. I have changed it to `Development Status :: 2 - Pre-Alpha`. Is that ok or shall I remove this line altogether?